.github/config/linux-evergreen.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I don't think this is fully hooked, I don't see it in the list of gha run—I believe you're missing the yaml code to hook it up

Reply:
done